timbermill-java/timbermill-server/src/test/java/com/datorama/oss/timbermill/TimberLogAdvancedTest.java

```diff
@@ -99,6 +99,26 @@ public void testOngoingTask() {
         assertEquals(string3, ongoingTask.getString().get(string3));
     }
 
+    public void testTaskReachedRateLimit() {
```

Comment:
Also add a test the after enough time has passed the timbermill will be able to process the limited tasks again